packages/core/src/transport/sendWithRetryStrategy.ts

```diff
@@ -1,12 +1,15 @@
 import { addTelemetryDebug } from '../domain/telemetry'
 import type { EndpointType } from '../domain/configuration'
 import { monitor } from '../tools/monitor'
+import type { RawError } from '../tools/error'
+import { clocksNow } from '../tools/timeUtils'
 import { ONE_KIBI_BYTE, ONE_MEBI_BYTE, ONE_SECOND } from '../tools/utils'
+import { ErrorSource } from '../tools/error'
 import type { Payload, HttpResponse } from './httpRequest'
 
 export const MAX_ONGOING_BYTES_COUNT = 80 * ONE_KIBI_BYTE
 export const MAX_ONGOING_REQUESTS = 32
-export const MAX_QUEUE_BYTES_COUNT = 3 * ONE_MEBI_BYTE
+export const MAX_QUEUE_MIB_COUNT = 3
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** I think it is a bit confusing to have limits in bytes and mebibytes.  ‏FMU, you changed this constant for message (to show `XMiB`). Maybe you could show `${MAX_QUEUE_BYTES_COUNT / ONE_MEBI_BYTE}` when formating the error message?